tests/unit/sagemaker/jumpstart/test_cache.py

```diff
@@ -194,6 +230,32 @@ def test_jumpstart_cache_get_header():
         "v3-classification-4'?"
     ) in str(e.value)
 
+    with pytest.raises(KeyError) as e:
+        cache.get_header(
+            model_id="ai21-summarize",
+            semantic_version_str="1.1.003",
+            model_type=JumpStartModelType.PROPRIETARY,
+        )
+    assert (
+        "Unable to find model manifest for 'ai21-summarize' with version '1.1.003'. "
+        "Visit https://sagemaker.readthedocs.io/en/stable/doc_utils/pretrainedmodels.html "
```

Comment:
have we updated the [doc utility](https://github.com/aws/sagemaker-python-sdk/blob/master/doc/doc_utils/jumpstart_doc_utils.py) of the PySDK to list the proprietary models?

If not, fix the error message and add it to the project plan please.

Reply:
I'll add to the doc utility today